systems/data-plan/package/pkg/server/package.go

```diff
@@ -61,46 +71,50 @@ func (p *PackageServer) GetByOrg(ctx context.Context, req *pb.GetByOrgPackageReq
 	return packageList, nil
 }
 func (p *PackageServer) Add(ctx context.Context, req *pb.AddPackageRequest) (*pb.AddPackageResponse, error) {
-	logrus.Infof("Add Package Name: %v, SimType: %v, Active: %v, Duration: %v, SmsVolume: %v, DataVolume: %v, Voice_volume: %v", req.Name, req.SimType, req.Active, req.Duration, req.SmsVolume, req.DataVolume, req.VoiceVolume)
-
-	orgID, err := uuid.Parse(req.GetOrgId())
+	orgID, err := uuid.FromString(req.GetOrgID())
 	if err != nil {
 		return nil, status.Errorf(codes.InvalidArgument,
 			"invalid format of org uuid. Error %s", err.Error())
 	}
 
 	_package := &db.Package{
-		Name:         req.GetName(),
-		Sim_type:     req.GetSimType().String(),
-		Org_id:       orgID,
-		Active:       req.Active,
-		Duration:     uint(req.GetDuration()),
-		Sms_volume:   uint(req.GetSmsVolume()),
-		Data_volume:  uint(req.GetDataVolume()),
-		Voice_volume: uint(req.GetVoiceVolume()),
-		Org_rates_id: uint(req.GetOrgRatesId()),
+		Uuid:        uuid.NewV4(),
+		Name:        req.GetName(),
+		SimType:     1,
```

Comment:
Using the type enum (instead of the number) will make it easier to update/debug in the future

Reply:
addressed